### DISCUSSION TITLE
使用flyio部屬遇到問題

### DISCUSSION THREAD

**Discussion by sogadisk** [libsgh/PanIndex#363] [Q&A]
==> Building image
WARN Failed to start remote builder heartbeat: must provide either an organization id or app name
WARN failed to create build in graphql: input:3: createBuild Could not find App

Error failed to fetch an image or build from source: error connecting to docker: must provide either an organization id or app name

這是執行最後LOG看到的錯誤

有使用 flyctl secrets set DATABASE_URL 替換後才部屬

**Comment by libsgh** [comment]
是按照`README.md`说明操作的吗？步骤六输入应用名称输入了？

**Reply by sogadisk** [reply]
後來找到原因了  在步驟六輸入完名稱後 flyctl會產生新的[fly.toml] 將原先的檔案覆蓋導致一些cnv設定及port設定歸為預設所致
